Assignment 3

cponceph opened the pull request:
Finally fixed the repo, sorry for the inconvenience

ocorcho commented on Assignment3/cponceph-x150426/Exercise5.txt:
```diff
@@ -0,0 +1,8 @@
+PREFIX db: <http://dbpedia.org/ontology/>
+
+SELECT distinct ?prop (COUNT(?c) AS ?count) WHERE {
```
It should be count (Distinct ?c), but accepted anyway

ocorcho replied:
Well, I prefer not to accept it, because I have just realised that this query cannot be evaluated. Have you actually run it?

cponceph commented:
I didn´t ran the last query assuming it was correct, I'm sorry. I just changed the x variable for its proper name and added distinct as you suggested to get the property count:

PREFIX db: <http://dbpedia.org/ontology/>
SELECT distinct ?prop (COUNT(distinct ?c) AS ?count) WHERE {
?instance a db:Politician .
?instance ?prop ?c .
FILTER (?prop != rdf:type)
}


prop | count
-- | --
dbpedia2:border | 2
dbpedia2:latMin | 1
dbpedia2:employer | 9
dbpedia2:war | 1
dbpedia2:reference | 3
dbpedia2:father'sName | 1
dbpedia2:year2start | 3
dbpedia2:vicepremier | 7
dbpedia2:gap | 1
dbpedia2:1y | 16
dbpedia:ontology/deathYear | 78


If you accept the pull request I'll update the file along my group's Hands On 1 pull request. Otherwise I'll just make another one.
Thank you!

rgcmme commented:
- The content of the README.md file in the hands-on directory is not the proper one.

cponceph commented:
Fixed both files and pushed the changes.
